apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Snippets/SpannerConnectionSnippets.cs

```diff
@@ -36,5 +58,251 @@ public void CreateConnection()
             Assert.Equal("my-instance", connection.SpannerInstance);
             Assert.Equal("my-db", connection.Database);
         }
+
+        [Fact]
+        public async Task CreateDatabaseAsync()
+        {
+            // CreateDatabaseAsync will create a new database and dispose of it right after.
+            // But for consistency sake, we want to show the same variables in the snippets.
+            _databaseName = "t_" + Guid.NewGuid().ToString("N").Substring(0, 28);
+
+            // Sample: CreateDatabaseAsync
+            // Additional: CreateDdlCommand
+            using (var connection =
+                new SpannerConnection($"Data Source=projects/{_projectId}/instances/{_instanceName}/databases/{_databaseName}"))
+            {
+                var createDbCmd = connection.CreateDdlCommand($"CREATE DATABASE {_databaseName}");
+                await createDbCmd.ExecuteNonQueryAsync();
+
+                var createTableCmd = connection.CreateDdlCommand(
+                    @"CREATE TABLE TestTable (
+                                            Key                STRING(MAX) NOT NULL,
+                                            StringValue        STRING(MAX),
+                                            Int64Value         INT64,
+                                          ) PRIMARY KEY (Key)");
+                await createTableCmd.ExecuteNonQueryAsync();
```

Comment:
What's the result of this for CREATE TABLE? Worth printing to the console? (Ditto for CREATE DATABASE.)

Reply:
You mean the return value of the execute?  Its ignorable.  you could query information_schema to get the table info afterwards, but this is meant to be a super simple example.  Maybe I'll add that later as another snippet.

Reply:
I was more genuinely intrigued as to what the result would be, but I don't think it's important for a snippet :)

Reply:
oh I think we return 0 here.